net/davix/CMakeLists.txt

```diff
@@ -9,18 +9,22 @@
 # @author Pere Mato, CERN
 ############################################################################
 
+if(iolite)
+  set(RIOLITELIBS ROOTIOLite)
+endif()
+
 ROOT_STANDARD_LIBRARY_PACKAGE(RDAVIX
   HEADERS
     ROOT/RRawFileDavix.hxx
     TDavixFile.h
     TDavixSystem.h
   SOURCES
-    src/RRawFileDavix
+    src/RRawFileDavix.cxx
     src/TDavixFile.cxx
     src/TDavixSystem.cxx
   LIBRARIES
+    ${RIOLITELIBS}
```

Comment:
I still think that this should come via RIO ... i.e. something else is 'broken' per se if that is not currently happening.  This important because we don't want to have all users of RIO have to add something similar.

Reply:
@pcanal  I hope you saw lines 12-14:
```
if(iolite)
  set(RIOLITELIBS ROOTIOLite)
endif() 
```

We need ${RIOLITELIBS} **only** when we build with `iolite` option which is OFF by default (e.g. building with `iolite` option means have ROOTIOLite without dictionary and PCM).

Reply:
Sure ... but still in this case, RIO would be linked against ROOTIOLite, right?

Reply:
I need to provide the possibility to support both cases: to be able to build with and without PCMs. Otherwise, there is no sense in having this library used as a standalone.

Reply:
btw I added WIP status because I can't reproduce error locally but only on build nodes. I am investigating.

Reply:
Yes but when the library exist then RIO depends on it, right?  If that the case, then there should never be a need to list it explicit in the libraries that already uses RIO.

If I understood some of the prior discussion this mean that for the declaration of RIO we need to have:
```
  DEPENDENCIES
    Core
    Thread
    ${RIOLITELIBS}
```
and (still If I understood correctly) this might require an upgrade `ROOT_LINKER_LIBRARY` to handle the case without pcm ... 

However maybe I am confused.  There is 4 possible config:

(a) no ROOTRioLite, i.e. current state per
(b) ROOTRioLite with pcm and RIO using it, then the code I wrote should work without any other change, right?
(c) ROOTRioLite **without** pcm and RIO using it, `ROOT_LINKER_LIBRARY` might need an upgrade.
(d) ROOTRioLite without pcm and RIO is not build.

Which config do you think we should support?  [I thought that we were aim for __only__ (b) and (d).]